anyway/parsers/compare_cbs_and_anyway_road_segments_accidents.py

```diff
@@ -0,0 +1,265 @@
+        df.columns = columns
+        df = df.loc[df[segment_col].notna() & df[segment_col].astype(str).str.isdigit()]
+        return df
+    except Exception as e:
+        print(f"Error reading {file_path}: {e}")
+        return pd.DataFrame()
+
+
+def get_cbs_count():
+    df_type_1_columns = [
+        "segment",
+        "road",
+        "from",
+        "to",
+        "acc_per_million_km",
+        "total",
+        "total_light",
+        "total_severe",
+        "total_fatal",
+        "2022_total",
+        "2022_light",
+        "2022_severe",
+        "2022_fatal",
+        "2021_total",
+        "2020_total",
+        "avg",
+        "length",
+    ]
+    df_type_3_columns = [
+        "segment",
+        "road",
+        "from",
+        "to",
+        "acc_per_million_km",
+        "total",
+        "2022_total",
+        "2021_total",
+        "2020_total",
+        "avg",
+        "length",
+    ]
+
+    # Read and process type 1 data
+    df_type_1 = read_excel_file(CBS_TYPE_1_SUMMARY_FILE, 4, df_type_1_columns)
+    if df_type_1.empty:
+        return pd.DataFrame()
+    df_type_1["provider_code"] = 1
+    df_type_1["road_segment_name_cbs"] = (
+        df_type_1["from"].str.slice(start=1) + " -" + df_type_1["to"].str.slice(start=2)
+    )
+    df_type_1_total = df_type_1[
+        [
+            "road_segment_name_cbs",
+            "road",
+            "segment",
+            "provider_code",
+            "2020_total",
+            "2021_total",
+            "2022_total",
+        ]
+    ].copy()
+    df_type_1_total.columns = [
+        "road_segment_name_cbs",
+        "road",
+        "segment",
+        "provider_code",
+        "2020_cbs",
+        "2021_cbs",
+        "2022_cbs",
+    ]
+    df_type_1_total.fillna({"2020_cbs": 0, "2021_cbs": 0, "2022_cbs": 0}, inplace=True)
+
+    # Read and process type 3 data
+    df_type_3 = read_excel_file(CBS_TYPE_3_SUMMARY_FILE, 5, df_type_3_columns)
+    if df_type_3.empty:
+        return df_type_1_total
+    df_type_3["provider_code"] = 3
+    df_type_3["road_segment_name_cbs"] = (
+        df_type_3["from"].str.slice(start=1) + " - " + df_type_3["to"].str.slice(start=2)
+    )
+    df_type_3_total = df_type_3[
+        [
+            "road_segment_name_cbs",
+            "road",
+            "segment",
+            "provider_code",
+            "2020_total",
+            "2021_total",
+            "2022_total",
+        ]
+    ].copy()
+    df_type_3_total.columns = [
+        "road_segment_name_cbs",
+        "road",
+        "segment",
+        "provider_code",
+        "2020_cbs",
+        "2021_cbs",
+        "2022_cbs",
+    ]
+    df_type_3_total.fillna({"2020_cbs": 0, "2021_cbs": 0, "2022_cbs": 0}, inplace=True)
+
+    # Combine type 1 and type 3 data
+    df_cbs_total = pd.concat([df_type_1_total, df_type_3_total])
+    df_cbs_total.set_index(["road", "segment", "provider_code"], inplace=True)
+    return df_cbs_total
+
+
+def get_anyway_count():
```

Comment:
Why not use `get_query()`, or `get_accidents_stats()`?

Reply:
@ziv17 
I wanted to have the option to modify location_accuracy values to generate different results.
However now that we've decided that 1,3,4 are the sub-urban resolutions, I can use the constants you created (Or use one of the functions you've suggested instead)